### PR TITLE
Re-enable Postgres tests and remove MariaDB

### PR DESCRIPTION
**Pull request by pavannd1**
This reverts commit 06be966f576ed850e705a9fc31a475a512a7d2bc.

Additionally, this PR removes MariaDB tests. We will add it back once we have enough resources to run all the tests.

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [x] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [x] :robot: Test

### CODE REVIEW

**Comment by pavannd1** [comment]
Looks like the mongo-db tests are failing now.

**Comment by akankshakumari393** [comment]
if we disable one of the app, the integration test works, I tried disabling MySQL, the test passed [commit](https://github.com/kanisterio/kanister/pull/2370/commits/7eb9ec49a785b5f94a3c7bea2b640b1c3ca75d16) . [https://github.com/kanisterio/kanister/actions/runs/6390938725/job/17345776852](https://github.com/kanisterio/kanister/actions/runs/6390938725/job/17345776852), maybe we should upgrade the test infrastructure.

**Comment by akankshakumari393** [comment]
I have reverted the commit, that I made in this PR to test the CI.

File: build/integration-test.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ TEST_TIMEOUT="30m"
 TEST_OPTIONS="-tags=integration -timeout ${TEST_TIMEOUT} -check.suitep ${DOP}"
 # Regex to match apps to run in short mode
 # Temporary disable ES test. Issue to track https://github.com/kanisterio/kanister/issues/1920
-SHORT_APPS="^MySQL$|^MongoDB$|Maria|^MSSQL$"
+SHORT_APPS="^PostgreSQL$|^MySQL$|^MongoDB$|Maria|^MSSQL$"
```
Comment (pavannd1) [inline build/integration-test.sh:29]:
```suggestion
SHORT_APPS="^PostgreSQL$|^MySQL$|^MongoDB$|^MSSQL$"
```

**Review by akankshakumari393 — APPROVED** [review]

**Pull request merged**